src/components/NewHero/NewHero.tsx

```diff
@@ -0,0 +1,66 @@
+import { useSelector } from 'react-redux';
+import getImageURL from '../../utils/getImageURL';
+import { GlobalStateType } from '../../utils/types';
+import {
+  HeroContainer,
+  MainHero,
+  SecondaryHero,
+} from './styles';
+
+export const BASE_IMAGE_URL = 'https://agenciadenoticias.ibge.gov.br/';
+
+function NewHero() {
+  const globalState = useSelector((state: GlobalStateType) => state.news);
+  const { breakingNews } = globalState;
```

Comment:
Se o `globalState` não vai ser utilizado aqui, é mais interessante que seja selecionado apenas a chave `breakingNews` para evitar renderizações desnecessárias, conforme sugerido na [style guide do redux](https://redux.js.org/style-guide/#call-useselector-multiple-times-in-function-components).

Uma sugestão também é separar o `selector function` em um arquivo diferente. Vou deixar antes do componente, mas a ideia é que ele esteja em outro lugar. Sugerido em [outro tópico da mesma style guide](https://redux.js.org/style-guide/#use-selector-functions-to-read-from-store-state).

```suggestion
const selectBreakingNews = (state: GlobalStateType) => state.news.breakingNews;

function NewHero() {
  const breakingNews = useSelector(selectBreakingNews());
```